test/cmd/images.sh

```diff
@@ -64,7 +64,7 @@ oc describe istag/mysql:latest
 [ "$(oc describe istag/mysql:latest | grep "Image Created:")" ]
 [ "$(oc describe istag/mysql:latest | grep "Image Name:")" ]
 name=$(oc get istag/mysql:latest --template='{{ .image.metadata.name }}')
-imagename="isimage/mysql@${name:0:7}"
+imagename="isimage/mysql@${name}"
```

Comment:
Mysql 55 and mysql 56.

Ben Parees | OpenShift
On Oct 28, 2015 12:23 AM, "Clayton Coleman" <notifications@github.com>
wrote:

> In test/cmd/images.sh
> <https://github.com/openshift/origin/pull/5409#discussion_r43217260>:
>
> > @@ -64,7 +64,7 @@ oc describe istag/mysql:latest
> >  [ "$(oc describe istag/mysql:latest | grep "Image Created:")" ]
> >  [ "$(oc describe istag/mysql:latest | grep "Image Name:")" ]
> >  name=$(oc get istag/mysql:latest --template='{{ .image.metadata.name }}')
> > -imagename="isimage/mysql@${name:0:7}"
> > +imagename="isimage/mysql@${name}"
>
> To verify that you can retrieve images by their short ID. Which we still
> want to do, but if there are two we should be able to choose the first. I'm
> not sure why there are 2 now though.
> … <#150acafb50312f2c_>
> On Tue, Oct 27, 2015 at 11:58 PM, Ben Parees <notifications@github.com>
> wrote: In test/cmd/images.sh <#5409 (comment)
> <https://github.com/openshift/origin/pull/5409#discussion_r43216441>>: >
> @@ -64,7 +64,7 @@ oc describe istag/mysql:latest > [ "$(oc describe
> istag/mysql:latest | grep "Image Created:")" ] > [ "$(oc describe
> istag/mysql:latest | grep "Image Name:")" ] > name=$(oc get
> istag/mysql:latest --template='{{ .image.metadata.name }}') >
> -imagename="isimage/mysql@${name:0:7}" > +imagename="isimage/mysql@${name}"
> @smarterclayton <https://github.com/smarterclayton> <
> https://github.com/smarterclayton> i'm not clear on why were were
> truncating this value previously? but it seems to be causing failures now
> that we have two valid images that start with mysql — Reply to this email
> directly or view it on GitHub <
> https://github.com/openshift/origin/pull/5409/files#r43216441>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5409/files#r43217260>.
>